CHANGELOG.md

```diff
@@ -38,6 +42,7 @@ _📚 Refer to the [GitHub Release Notes] for previous changelogs._
 
 <!-- PR links -->
 
+[#567]: https://github.com/munich-quantum-toolkit/ddsim/pulls/567
 [#571]: https://github.com/munich-quantum-toolkit/ddsim/pulls/571
```

Comment:
```suggestion
[#571]: https://github.com/munich-quantum-toolkit/ddsim/pulls/571
[#567]: https://github.com/munich-quantum-toolkit/ddsim/pulls/567
```
I'd have a slight tendency to order the PRs here chronologically with the highest number coming first. Creates a bit of order 😌